spec/blocksync/readme.md

```diff
@@ -0,0 +1,64 @@
+---
+order: 
+parent:
+    title: blocksync
+    order: 
+---
+
+
+In a proof of work blockchain, syncing with the chain is the same process as staying up-to-date with the consensus: download blocks, and look for the one with the most total work. In proof-of-stake, the consensus process is more complex, as it involves rounds of communication between the nodes to determine what block should be committed next. Using this process to sync up with the blockchain from scratch can take a very long time. It's much faster to just download blocks and check the merkle tree of validators than to run the real-time consensus gossip protocol.
+
+The Blocksync Reactor's high level responsibility is to enable peers who are
```

Comment:
Are we defining a _reactor_  here, which is an implementation name, or a _service_. For the sake of specification, I would prefer to define a service, which is more abstract, to then describe an implementation of the service (e.g., as a `Reactor`).